uint32 id
string name
string emotion
string gender
sensor_msgs/RegionOfInterest bounding_box
